Obsidian/Net/MinecraftStream.Writing.cs

```diff
@@ -396,15 +396,15 @@ public void WritePosition(Position value)
         }
 
         [WriteMethod, Absolute]
-        public void WriteAbsolutePosition(Position value)
+        public void WriteAbsolutePosition(API.Vector value)
```

Comment:
Why is it API.Vector here, but just VectorF in other places?

Reply:
This was done by VS for some reason.

Reply:
Explicit namespaces either mean a missing import or conflicting member names.